src/core/dev/net_device_val.cpp

```diff
@@ -931,6 +931,7 @@ bool net_device_val::update_active_slaves()
     return 0;
 }
 
+// IFTAH - azure related...
```

Comment:
Just change to DOCA TODO instead of 'IFTAH'. and below

Reply:
Done